.devcontainer/Dockerfile

```diff
@@ -1,7 +1,5 @@
-#-------------------------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
-#-------------------------------------------------------------------------------------------------------------
+# Licensed under the MIT License.
 
```

Comment:
The new copyright scanner is stricter.  (This file was one of two Dockerfiles that had a 'unique' header.)